example/spec/parallel/asyncOperations/asyncOperationRunnerSpec.js

```diff
@@ -16,20 +16,22 @@ describe('The AsyncOperation task runner', () => {
   let asyncOperationModel;
   let config;
   let cluster;
+  let asyncOperationsTableName;
   let asyncOperationTaskDefinition;
   let successFunctionName;
   let failFunctionName;
 
   beforeAll(async () => {
     config = await loadConfig();
 
+    asyncOperationsTableName = `${config.stackName}-AsyncOperationsTable`;
```

Comment:
Like the windmill said, "I'm really not a big fan."

Looking at these tests, I think that they belong in the API package and not `example`, as they're lower-level integration tests and not the "test Cumulus using its interfaces" tests, which I think is what the `example` tests should be. I'll issue a PR to fix that.

Reply:
Note: The thing that made this feel wrong was that our end-to-end test 1) was making an assumption about the name of the AsyncOperationsTable and 2) needed to know that table name at all. If it's trying to do something that can't be done through the API, then it probably doesn't belong in the end-to-end tests.

Another note: Yes, I realize that I wrote these test a while back. I was young and naive.

Reply:
> was making an assumption about the name of the AsyncOperationsTable

Don't most of our integration tests do this?

Reply:
I'd hope not but, even if they do, I'd rather not do more of it.

Reply:
> Like the windmill said, "I'm really not a big fan."

That makes two of us.

> I'll issue a PR to fix that.

I'm all for having the optimal solution, but this is impacting terraform CI, so do you plan for that to be done soon or can we consider merging this in the meantime with the assumption that it will be completely refactored out?

Reply:
Yeah, we can refactor it later.